gapis/api/transform2/transform.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transform2
+
+import (
+	"context"
+
+	"github.com/google/gapid/gapis/api"
+)
+
+// Transform is the interface that wraps the basic Transform functionality.
```

Comment:
This doc needs to be changed. 
Especially for something this important, you should have at least a few lines describing what the transform is/does/how its used /etc.
What is the "Basic Transform Functionality" .

Reply:
Agreed!

Reply:
For both this and Writer I have moved them from the old file(transformer.go) directly and in the old file, the explanation was spread into comments for each function. Which I have tried to keep the same approach. Which means that the real explanation is the comment to the function `TransformCommand` We may still expand the definition as well.